new/PRS13146SaggaKrestos.xml

```diff
@@ -38,16 +38,16 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
             </langUsage>
         </profileDesc>
         <revisionDesc>
-            <change who="DE" when="2020-09-14">Created entity</change>
+            <change who="MKr" when="2020-09-14">Created entity</change>
             <change who="MKr" when="2020-09-14">Completed description</change>
         </revisionDesc>
     </teiHeader>
     <text>
         <body>
             <listPerson>
                 <person sex="1">
-                    <persName xml:lang="gez" xml:id="n1">Ṣaggā Krǝstos</persName>
-                    <persName xml:lang="gez" type="normalized" corresp="#n1">ጸጋ፡ ክርስቶስ፡</persName>
+                    <persName xml:lang="gez" type="normalized" xml:id="n1">Ṣaggā Krǝstos</persName>
+                    <persName xml:lang="gez" corresp="#n1">ጸጋ፡ ክርስቶስ፡</persName>
                     <faith type="EOTC"/>
```

Comment:
xml:id comes with the name in Fidal, and the transliteration gets a corresp in this case